src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
```

Comment:
The class `MainApp` should not have redundant empty lines after the class declaration. Please remove the empty line to follow best practices.